apps/web/src/routes/MapExplorer/MapExplorer.js

```diff
@@ -695,8 +695,6 @@ function MapExplorer (props) {
   const { hideNavLayout } = layoutFlags
   const withoutNav = isWebView() || hideNavLayout
 
-  const locationParams = location !== undefined ? getQuerystringParam(['zoom', 'center', 'lat', 'lng'], location) : null
```

Comment:
Where did these go?

Reply:
the new useViewHeaderDetails hook just pulls in all existing query params into it. thats all this was being used for